### DISCUSSION TITLE
After upgrading filament showing error. Please help me.

### DISCUSSION THREAD

**Discussion by sajjadhossainshohag** [filamentphp/filament#3233] [Help] [answered]
( Cannot use object of type Filament\Navigation\NavigationGroup as array) this type of error after the upgrade.

**Comment by danharrin** [comment, marked as answer]
1) Run `php artisan filament:upgrade`, this should be in your update script

2) If you have a `/resources/views/vendor/filament` folder, delete it. Overriding views is not recommended when using Filament.
